### PR TITLE
SELC-2288 - added environment modal for products other than Interop

### PR DESCRIPTION
**Pull request by esignore**

### CODE REVIEW

File: src/components/DashboardHeader.tsx [hunk 1]
```diff
@@ -31,7 +31,9 @@ const DashboardHeader = ({ onExit, loggedUser, parties }: Props) => {
   const products = useAppSelector(partiesSelectors.selectPartySelectedProducts);
   const selectedParty = useAppSelector(partiesSelectors.selectPartySelected);
 
-  const [openEnvironmentModal, setOpenEnvironmentModal] = useState<boolean>(false);
+  const [openCustomEnvInteropModal, setopenCustomEnvInteropModal] = useState<boolean>(false);
```
Comment (kaiban) [inline src/components/DashboardHeader.tsx:34]: setter function name should be in camelcase

File: src/pages/dashboardOverview/components/activeProductsSection/components/ActiveProductCardContainer.tsx [hunk 2]
```diff
@@ -23,7 +24,8 @@ export default function ActiveProductCardContainer({
   const { t } = useTranslation();
   const { invokeProductBo } = useTokenExchange();
 
-  const [openEnvironmentModal, setOpenEnvironmentModal] = useState<boolean>(false);
+  const [openCustomEnvInteropModal, setopenCustomEnvInteropModal] = useState<boolean>(false);
```
Comment (kaiban) [inline src/pages/dashboardOverview/components/activeProductsSection/components/ActiveProductCardContainer.tsx:27]: setter function name should be in camelcase

File: src/pages/dashboardOverview/components/activeProductsSection/components/ActiveProductCardContainer.tsx [hunk 3]
```diff
@@ -23,7 +24,8 @@ export default function ActiveProductCardContainer({
...
+  const [openGenericEnvProductModal, setopenGenericEnvProductModal] = useState<boolean>(false);
```
Comment (kaiban) [inline src/pages/dashboardOverview/components/activeProductsSection/components/ActiveProductCardContainer.tsx:28]: setter function name should be in camelcase

File: src/components/DashboardHeader.tsx (hunk 1 above)
Comment (esignore) [reply src/components/DashboardHeader.tsx]: done

File: src/pages/dashboardOverview/components/activeProductsSection/components/ActiveProductCardContainer.tsx (hunk 2 above)
Comment (esignore) [reply src/pages/dashboardOverview/components/activeProductsSection/components/ActiveProductCardContainer.tsx]: done

**Review by kaiban — APPROVED** [review]

**Pull request merged**